test/studies/ssca2/graphio/SSCA2_RMAT_graph_generator.chpl

```diff
@@ -110,9 +110,9 @@ proc Gen_RMAT_graph ( a : real,
     // Random Numbers return in the range [0.0, 1.0)
 
     var Rand_Gen = if REPRODUCIBLE_PROBLEMS then
```

Comment:
I don't think Rand_Gen is deleted.  Can it be borrowed?

Reply:
Can't be borrowed, because that runs into if-expr issues (that @benharsh is working on fixing).

Reply:
I made it owned and the test seems to pass, so we'll go with that for now